region/brinstar/pink/Pink Brinstar Power Bomb Room.json

```diff
@@ -482,6 +493,20 @@
         "Alternatively, morph can help with the second hopper, as it won't be able to hit Samus."
       ]
     },
+    {
+      "link": [1, 4],
+      "name": "Use Flash Suit, Diagonal Spark on Entry",
+      "requires": [
+        {"useFlashSuit": {}},
+        {"shinespark": {"frames": 2, "excessFrames": 2}}
```

Comment:
Is this an echoes kill?  Sparking through a hopper without any energy will more likely get you killed.